src/Models/Properties/ElementProperties/BaseProperty.cs

```diff
@@ -76,5 +76,11 @@ public partial class BaseProperty
         public string SummaryLabel { get; set; } = string.Empty;
 
         public string Warning { get; set; } = string.Empty;
+
+        public string OptionalIfQuestionId { get; set; } = string.Empty;
+
+        public string OptionalIfValue { get; set; } = string.Empty;
+
+        public string OptionalIfNotValue { get; set; } = string.Empty;
```

Comment:
Could/should these be extracted out to their own partial class? Like "RequiredIfProperty"... seems like the right thing to do.